src/lara-plugin/plugins/plugins.ts

```diff
@@ -45,6 +45,13 @@ export const initPlugin = (label: string, options: IPluginContextOptions) => {
 };
 
 const initRuntimePlugin = (label: string, options: IPluginRuntimeContextOptions) => {
+  // tslint:disable-next-line:no-console
+  console.info("initRuntimePlugin called with label: ", label);
+  if (!pluginClasses[label]) {
+    // tslint:disable-next-line:no-console
+    console.error("No pluginClasses label.");
```

Comment:
it is probably good to remove the rest of this console.log statements. But i'd keep this check and this console.error. That way an issue in the plugin system won't break the whole app, and we should still see the error in the console. 
I'd change the error message to include the label that couldn't be found. so maybe something like:
``console.error(`initRuntimePlugin cannot find plugin class with label: ${label}, plugin instance is not loaded`);``